libs/features/mas/src/checkout.js

```diff
@@ -49,13 +49,20 @@ export function Checkout({ settings, providers }) {
             entitlement,
             upgrade,
             modal,
+            fallbackStep,
```

Comment:
why do we need to introduce fallbackStep? in merch.js, we already know if the checkout link has a custom workflow step. if not, you can pass this value as workflow step.

Reply:
In our case the workflow step for the 3in1 link is different from the one that is used in the fallback URL.
For 3in1, the workflow step is always `segmentation`, and for the fallback, it may be `email`, `commitment` etc. So the `workflowStep` is used for 3in1 links, and `fallbackStep` is a workflow step used for fallback links.